weblate_web/templates/index.html

```diff
@@ -52,7 +52,12 @@ <h2 class="section-title hp-style">{% trans "Robust feature set" %}</h2>
                 <a class="button inline border rev" href="{% url 'features' %}">{% trans "Explore all features" %}</a>
             </div>
             <div class="wrap supporters">
-                <h2 class="section-title hp-style">{% trans "Users and Supporters" %}</h2>
+                <h2 class="section-title hp-style">{% trans "Companies using Weblate" %}</h2>
+                {% include "snippets/reviews.html" %}
+            </div>
+
+            <div class="wrap supporters">
+                <h2 class="section-title hp-style">{% trans "SUPPORTERS & OPEN SOURCE PROJECTS" %}</h2>
```

Comment:
Please no uppercase in the texts, it is handled by the CSS.

Reply:
Okay, it's been lowered.